.changeset/flat-horses-worry.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
🤔 Hmmm, I think there should only be a single changeset here, not three. Please revise so we only have one changeset!

On another note, I don't think this should be a major version. It adds to the API surface in a backwards compatible way from what I can tell, so a minor version bump would be fine from my perspective.

Reply:
How can I do the minor bump? In CLI I could only select major one.

Reply:
It's a touch confusing. Changeset will prompt you first if any selected packages should get a **major** bump. Don't select any packages, just hit Enter. You'll then get prompted for which packages should get a **minor** bump. This is where you can select `next-urql`.

<img width="671" alt="image" src="https://user-images.githubusercontent.com/19421190/93822432-167c9e80-fc15-11ea-9a3f-34f21a89fc69.png">

Reply:
Thanks for explanation :)